package.json

```diff
@@ -18,6 +18,7 @@
     "del": "^3.0.0",
     "file-loader": "^0.11.2",
     "font-awesome": "^4.7.0",
+    "fs": "^0.0.1-security",
```

Comment:
For some reason gulp doesn't recognize fs as a built in library so i had to added manually, 
It is added to dev dependancies yaml-js needs it, for some reason yml files are not read directly using require i assume support